tests/unit/draw/DefaultDrawer.spec.js

```diff
@@ -1085,20 +1085,22 @@ describe('Test Class: DefaultDrawer()', () => {
   });
 
   describe('Test method: arrangeComponentsPosition', () => {
```

Comment:
Move all modification of `Update tests` commit with commit that modified DefaultDrawer and other.

Reply:
done